mappings/net/minecraft/entity/ai/brain/LivingTargetCache.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_6670 net/minecraft/entity/ai/brain/LivingTargetCache
+	FIELD field_35104 EMPTY Lnet/minecraft/class_6670;
+	FIELD field_35105 entities Ljava/util/List;
+	FIELD field_35106 targetPredicate Ljava/util/function/Predicate;
+	METHOD <init> (Lnet/minecraft/class_1309;Ljava/util/List;)V
+		ARG 1 owner
+		ARG 2 entities
+	METHOD method_38971 empty ()Lnet/minecraft/class_6670;
+	METHOD method_38972 contains (Lnet/minecraft/class_1309;)Z
```

Comment:
Should this be `canTarget` or just `contains`, as this first checks if the entity is in the list and then checks if it matches the target predicate?

Reply:
Since all operations in this class test the target predicate, I think this class represents a list of entities that only match the predicate.